lib/handler.ex

```diff
@@ -50,12 +50,20 @@ defmodule SAXMap.Handler do
     {:ok, {[current | stack], options}}
   end
 
-  def handle_event(:characters, "\r" <> _, state) do
-    {:ok, state}
+  def handle_event(:characters, "\r" <> chars, state) do
```

Comment:
```suggestion
  def handle_event(:characters, "\r" <> _ = chars, state) do
```

Keep the same process as "\n" case.